internal/store/postgres/migrations/000020_add_constraint_email_unique.up.sql

```diff
@@ -0,0 +1,11 @@
+DO $$
```

Comment:
we don't drop the uuid column on the migration?

Reply:
Oh forgot to add that, lol although that's our goal haha. Thanks mas for notice it